Remove superagent-retry plugin

rifflock opened the pull request:
segmentio/analytics-node#100

vadimdemedes commented:
Thanks for this PR! Only few things left to get it merged:

1. Remove changes to `analytics-node.js` and `yarn.lock` from PR
2. Remove https://github.com/segmentio/analytics-node/blob/master/lib/index.js#L153

rifflock commented:
Okay, I've updated my commit to remove the changes to `analytics-node.js` and `yarn.lock`

I removed the call to `request.retry`, but I'm curious why that should be removed since the `retry` method is supported by `superagent` by default.

vadimdemedes commented:
> I removed the call to `request.retry`, but I'm curious why that should be removed since the `retry` method is supported by `superagent` by default.

My bad, I missed the fact that `superagent` has `retry()` by default now. Could you bring this line back?

rifflock commented:
Done and done!

vadimdemedes commented:
Thanks!

vadimdemedes commented:
Waiting on the build to turn green and merging 😉

vadimdemedes commented:
Could you please push any changes to this branch, so that CI is triggered? The reason why this is needed is because CI was enabled after you pushed your last commit and I can't run the build manually for this PR.

What needs to be done is simple:

1. Add any meaningless change and push it to this branch
2. Revert this commit and force push

So nothing needs to change in this PR, it's only to trigger the build.

vadimdemedes commented:
Weird, I don't see any builds created for this PR. Could you create a new PR for it? Sorry that you have to go through all that trouble.

Steps to do so:

```
$ git checkout use-builtin-retry-from-superagent
$ git checkout -b remove-superagent-retry
$ git push origin head
```
